Fix SDK subscription message handling (#27859)
directus/directus: .changeset/sdk-realtime-buffer-init.md

---
'@directus/sdk': patch
---

> Fixed a race in realtime subscriptions where the first message received could be either the `init` message or the first event, depending on timing.

Reviewer: Nit

```suggestion
Fixed a race condition in realtime subscriptions that could drop the initial message
```
